pachyderm/templates/pachd/deployment.yaml

```diff
@@ -2,6 +2,7 @@
 SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
 SPDX-License-Identifier: Apache-2.0
 */ -}}
+{{- if .Values.pachd.enabled }}
```

Comment:
I’d be tempted to do `{{- if and .Values.pachd .Values.pachd.enabled }}`.

Reply:
`pachd` should always exist as it has values in values.yaml, but the point is well taken for other instances.